browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2320,21 +2320,13 @@ L.Control.JSDialogBuilder = L.Control.extend({
 				var label = L.DomUtil.create('label', 'ui-content unolabel', button);
 				label.htmlFor = buttonId;
 				label.textContent = builder._cleanText(data.text);
-				button.setAttribute('alt', label.textContent);
```

Comment:
no tooltip data in this case?

Reply:
I was focused on buttons first wave, I have to change other places too. Thanks, if you know of other places too, it will help

Reply:
I think it is important. Screen readers will read "alt" for images. Even if we add new tooltip it still has to be present I belive. In other case we will not pass automatic a11y checkers.

Reply:
I checked with Firefox a11y checker and it seems to be valid issue:

![a11y](https://github.com/CollaboraOnline/online/assets/5307369/6b1fffa6-2e4a-4cfa-8346-320535c9b26c)

Reply:
> I think it is important. Screen readers will read "alt" for images. Even if we add new tooltip it still has to be 
> present I belive. In other case we will not pass automatic a11y checkers.

It will conflict with native tooltips browser. Please write your concerns in the task description, it is not the scope of my task.

Reply:
> I checked with Firefox a11y checker and it seems to be valid issue:

Also, it is not my scope of my task. Please file a new issue in the accessibility task.

Reply:
Thank you for the fix, now it will work as before